packages/shared-ux/code_editor/impl/react_monaco_editor/types.ts

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Can we follow the `@notice` pattern, example in https://github.com/elastic/kibana/blob/main/kbn_pm/src/lib/normalize_path.mjs#L3-L28

After we'll have to regenerate the notice file with `node scripts/notice`

Reply:
Thanks, so I understand with this approach I can simplify and don't do a separate header with linting and license file? will do, thanks